Client/Frontend/Components/FakespotSettingsCardView.swift

```diff
@@ -0,0 +1,100 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+import Shared
+
+final class FakespotSettingsView: UIView, ThemeApplicable {
+    private struct UX {
+        static let headerLabelFontSize: CGFloat = 17
+        static let buttonLabelFontSize: CGFloat = 16
+        static let buttonCornerRadius: CGFloat = 14
+        static let contentStackViewSpacing: CGFloat = 16
+        static let labelSwitchStackViewSpacing: CGFloat = 12
+        static let contentInsets = UIEdgeInsets(top: 8, left: 0, bottom: 8, right: 0)
+        static let buttonInsets = UIEdgeInsets(top: 12, left: 16, bottom: 12, right: 16)
+    }
+
+    var onSwitchValueVhanged: ((Bool) -> Void)?
+
+    private lazy var contentStackView: UIStackView = .build { stackView in
+        stackView.axis = .vertical
+        stackView.spacing = UX.contentStackViewSpacing
+        stackView.isLayoutMarginsRelativeArrangement = true
+        stackView.layoutMargins = UX.contentInsets
+    }
+
+    private lazy var labelSwitchStackView: UIStackView = .build { stackView in
+        stackView.alignment = .fill
+        stackView.spacing = UX.labelSwitchStackViewSpacing
+    }
+
+    private lazy var headerLabel: UILabel = .build { label in
+        label.setContentCompressionResistancePriority(.required, for: .vertical)
+        label.text = Shopping.SettingsCardRecommendedProductsLabel
+        label.numberOfLines = 0
+        label.adjustsFontForContentSizeCategory = true
+        label.clipsToBounds = true
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
+                                                            size: UX.headerLabelFontSize)
+    }
+
+    private lazy var `switch`: UISwitch = .build { uiSwitch in
```

Comment:
Why do we need \` around this variable name? Can we have a name that doesn't have those \`?